src/index.html

```diff
@@ -10,7 +10,28 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class ="search-form search-form_big" data-qa="big">
+      <label class ="search-form__label search-form__label_big">
+        <input
+          data-qa ="keypress"
+          class= "search-form__input  search-form__input_big"
+          type ="text"
+          placeholder ="Try &quot;Los Angeles&quot;"
+        >
```

Comment:
Don't forget to close the input tag.

Reply:
input here is fine, habit from React on Darek's part most likely (in jsx you add the closing tag for inputs)